fsm.cpp

```diff
@@ -9,9 +9,16 @@
 fsm::stateHandler_t fsm::stateHandler = fsm::state0_idle;
 unsigned int lastStateHandleTime = millis();
 
-static unsigned int COUNTDOWN_TIMER = 15000;
-static unsigned int DURATION_TIMER = 30000;
-static unsigned int STILLNESS_TIMER = 15000;
+const unsigned int MAX_COUNTDOWN_TIMER = 20000; // 20s
+const unsigned int MIN_DURATION_TIMER = 30000; // 30s
+const unsigned int MIN_STILLNESS_TIMER = 30000; // 30s
```

Comment:
I think that max values would be better for this. Max values will prevent int overflow, which is good. 

Whereas, we want to be able to set the values to less than 30 seconds if we're in testing mode.

Reply:
I believe from our discussion, we concluded that overflows are not possible. Sampath initially recommended having min bounds for these values so we cannot accidentally set them to a value lower than 'normal'. However, I've message him about this, as we should allow for lower values for our Brave/client system testing.